frontend/src/metabase/nav/components/CollectionBreadcrumbs/CollectionBreadcrumbs.tsx

```diff
@@ -13,21 +17,48 @@ import {
 
 export interface CollectionBreadcrumbsProps {
   collection?: Collection;
+  onClick?: (collection: CollectionEssentials) => void;
+  baseCollectionId: CollectionId | null;
+}
+
+function getCollectionList({
```

Comment:
Feel free to suggest some better logic here. I'm just trying to get things working but I know this can be better.